azure-pipelines.yml

```diff
@@ -151,14 +137,15 @@ stages:
               shareWith: "$(itOrg)"
               cwd: "$(System.ArtifactsDirectory)"
 
+          # Install extension
           - task: InstallAzureDevOpsExtension@3
             displayName: "Install extension in IT org"
             inputs:
               connectTo: "VsTeam"
               connectedServiceName: "AzDo Extension - Marketplace - IntegrationTests"
               method: "id"
               publisherId: "$(publisher)"
-              extensionId: "sonarcloud"
+              extensionId: "sonarcloud-test"
```

Comment:
as long as we have one org, we need this to keep the extensionID different right?

Reply:
agreed and done